### DISCUSSION TITLE
I have a problem using @default(uuid()) with mysql

### DISCUSSION THREAD

**Discussion by sctnightcore** [prisma/prisma#11747] [Q&A] [answered]
I have a problem using @default(uuid()) with mysql

````
  Id         String       @id @default(uuid())
````
to
```
    `Id` VARCHAR(191) NOT NULL,
```

Why doesn't the default have uuid() ?

```
    "@prisma/client": "^3.9.2",
    "prisma": "^3.9.2",

```

**Comment by janpio** [comment, marked as answer]
Because that is a Prisma level default. When you create a value using Prisma Client, a UUID will be inserted.

**Reply by andreconghau** [reply]
nice explain. Thanks!~

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
